user_docs/en/changes.md

```diff
@@ -43,6 +43,7 @@ The available options are:
   * It is now possible to use braille display routing keys to move the text cursor. (#9101)
   * It is now possible to use the review cursor selection commands to select text.
 * Updating NVDA while add-on updates are pending no longer results in the add-on being removed. (#16837)
+* It is possible to interact with Data validation dropdown lists in Microsoft Excel 2016 and above. (#15138)
```

Comment:
Given my tests, it seems that such interactions were already possible with Excel 2016.